Week1/homework/todo.sql

```diff
@@ -0,0 +1,29 @@
+
+
+CREATE SCHEMA IF NOT EXISTS `elmira-DB` DEFAULT CHARACTER SET utf8 ;
```

Comment:
Be careful when creating a PR that it only has the things you want to be reviewed. This looks like a decent schema you've created, but you probably didn't need it for the week1 homework.